roles/kafka_connect_replicator/tasks/rbac_replicator_consumer.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: Get Authorization Token for Connect Replicator Consumer ERP User
+  import_role:
+    name: common
+    tasks_from: get_authorization_token.yml
+  vars:
+    oauth: "{{ kafka_connect_replicator_oauth_enabled }}"
+    oauth_user: "{{ kafka_connect_replicator_consumer_erp_oauth_user }}"
+    oauth_password: "{{ kafka_connect_replicator_consumer_erp_oauth_password }}"
+    ldap_user: "{{ kafka_connect_replicator_consumer_erp_admin_user }}"
+    ldap_password: "{{ kafka_connect_replicator_consumer_erp_admin_password }}"
+  when: kafka_connect_replicator_rbac_enabled or kafka_connect_replicator_oauth_enabled
```

Comment:
shouldn't this be kafka_connect_replicator_consumer....?

Reply:
the <component>_oauth_enabled variables are only present for components where we have the server level support for oauth which is in connect_replicator. Since replicator_consumer, replicator_producer and replicator_interceptor are only oauth clients and not servers I havent created such vars for them

Reply:
but we do have `kafka_connect_replicator_consumer_rbac_enabled` variable

Reply:
I think it will be best to use `kafka_connect_replicator_consumer_rbac_enabled` or `kafka_connect_replicator_oauth_enabled` here

Reply:
or we need to create _oauth_enabled vars for all vars corresponding to _rbac_enabled, let's discuss this later

Reply:
oh okay i missed it. agree that `kafka_connect_replicator_consumer_rbac_enabled or kafka_connect_replicator_oauth_enabled` should be used here.

We can discuss on `_rbac_enabled` as well but I think not using it will be better